app/views/fines/_fines_table.html.erb

```diff
@@ -0,0 +1,28 @@
+<p>NOTE: Fines for items that are still checked out will not appear here until the items have been returned</p>
+<table class="table">
+  <caption>Bills Table</caption>
```

Comment:
```suggestion
  <caption>Bills</caption>
```

or maybe we even do not need the caption since there is also `<h1>Bills</h1>` ?

Reply:
I think this is outside the topic of what the issue is here. Maybe let's let #2 happen and make one sweep through after that is what I'm thinking. Thoughts?

Reply:
sounds good 👍